omod/src/main/webapp/resources/scripts/service/tagService.js

```diff
@@ -0,0 +1,47 @@
+angular.module('tagService', ['ngResource'])
+    .factory('Tag', function($resource) {
+        return $resource("/" + OPENMRS_CONTEXT_PATH  + "/ws/rest/v1/tag/:uuid", {
+            uuid: '@uuid'
+        },{
+            query: { method:'GET', isArray:false },
```

Comment:
I don't think you need to add `isArray:false`, if it doesn't work you probably, in fact need to add the `uicommons.common` angular module as a dependency (NOT JAVA)